ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts

```diff
@@ -476,14 +476,7 @@ describe('AddressEmbeddedComponent', () => {
     component.ngOnInit();
     fixture.detectChanges();
 
-    expect(childComponent.country).toBeNull();
-    component.activityGuid = '123';
-    component.country = 'CA';
-    tempAddress = new Address({country: 'CA'});
-    addressServiceSpy.getTempAddress.and.returnValue(of(tempAddress));
-    component.ngOnInit();
-    fixture.detectChanges();
-    expect(childComponent.country).toBe('CA');
+    expect(childComponent.country).toBe('US');
```

Comment:
I changed behavior. I was originally allowing for country in input form to show up if we get a full address that had a country that was different from the default country. For the sake of simplicity, I dropped that capability, so this chunk of test was no longer valid.